app/index.html

```diff
@@ -448,6 +591,7 @@
     <!-- build:js(.) scripts/vendor.js -->
     <!-- bower:js -->
     <script src="/bower_components/jquery/dist/jquery.js"></script>
+    <script src="/bower_components/slider-pro/dist/js/jquery.sliderPro.js"></script>
```

Comment:
minified version was installed through bower and is present inside plugin folder at bower_components. However, seeing as how these lines were auto generated by bower I'm not sure manually editing this to use the min version would be the best option.
Is there some production flag I can use with bower?
Same problem is occurring with jquery.